client/src/components/desk/returns/ReturnOptions.tsx

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react';
+import {Button, Popup} from "semantic-ui-react";
+
+export function ReturnOptions({error}: any) {
+    return (
+        <>
+            <Popup content={"hello"} trigger={<Button basic color={"red"}>Lost</Button>}/>
```

Comment:
is this a stub?

Reply:
ghost of christmas past, I need to delete it

Reply:
The real reason is at one point, I was going to have 2 buttons for returning requests, one to mark it returned and one to mark it lost/damaged.  So this was a popup that would open with 2 buttons inside, Lost and Damaged.  I quickly realized that this strategy doesn't work well with Photo ID returns since participants get their photo ID back for lost/damaged items.

That lead me to the new flow where you have one button and a dropdown for marking the item lost/damaged, but it defaults to marking them returned.